<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="settings_button">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>400</y>
      <width>301</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1430</x>
      <y>710</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>log_out-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="upload_button">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>265</y>
      <width>301</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Upload New Video</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>430</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>settings-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>0</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo-removebg-preview (1).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>160</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>home-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Logout_button">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>670</y>
      <width>301</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>290</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>upload-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="home_button">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>130</y>
      <width>301</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QLabel" name="drop_label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>48</y>
      <width>500</width>
      <height>250</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px dashed #aaa;
    border-radius: 10px;
    background-color: #f9f9f9;
    color: #555;
    font-size: 16px;
    qproperty-alignment: AlignCenter;
</string>
    </property>
    <property name="text">
     <string>Drag File Here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseFileButton">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>328</y>
      <width>350</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#chooseFileButton {
    background-color: #222;
    color: white;
    padding: 10px 20px;
    border: none;
    border-radius: 6px;
    font-size: 16px;
    font-weight: bold;
}

QPushButton#chooseFileButton:hover {
    background-color: #333;
}

QPushButton#chooseFileButton:pressed {
    background-color: #111;
}
</string>
    </property>
    <property name="text">
     <string>Choose File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>300</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Or</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="uploadProgressBar">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>704</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="uploadButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>700</y>
      <width>300</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#uploadButton {
    background-color: #4CAF50; /* ירוק מודרני */
    color: white;
    padding: 12px 24px;
    border: none;
    border-radius: 8px;
    font-size: 16px;
    font-weight: bold;
    text-transform: uppercase;
    letter-spacing: 1px;
}

QPushButton#uploadButton:hover {
    background-color: #45a049;
}

QPushButton#uploadButton:pressed {
    background-color: #3e8e41;
}</string>
    </property>
    <property name="text">
     <string>Upload</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>720</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>upload_button-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>668</x>
      <y>145</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>file_button-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="title_text">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>410</y>
      <width>550</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #ccc;
    border-radius: 15px;
    padding: 6px 10px;
    background-color: white;
</string>
    </property>
    <property name="placeholderText">
     <string>Enter Title</string>
    </property>
   </widget>
   <widget class="QLabel" name="thumb_label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>410</y>
      <width>381</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>no_thumb.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="change_thumbnail">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>630</y>
      <width>261</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#change_thumbnail {
    background-color: #222;
    color: white;
    padding: 10px 20px;
    border: none;
    border-radius: 6px;
    font-size: 16px;
    font-weight: bold;
}

QPushButton#change_thumbnail:hover {
    background-color: #333;
}

QPushButton#change_thumbnail:pressed {
    background-color: #111;
}
</string>
    </property>
    <property name="text">
     <string>Change Thumbnail</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="description_text">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>480</y>
      <width>550</width>
      <height>200</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="placeholderText">
     <string>Enter Description</string>
    </property>
   </widget>
   <widget class="QLabel" name="error_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>Logout_button</zorder>
   <zorder>settings_button</zorder>
   <zorder>label_4</zorder>
   <zorder>upload_button</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label</zorder>
   <zorder>home_button</zorder>
   <zorder>label_5</zorder>
   <zorder>drop_label</zorder>
   <zorder>chooseFileButton</zorder>
   <zorder>label_7</zorder>
   <zorder>uploadProgressBar</zorder>
   <zorder>uploadButton</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>title_text</zorder>
   <zorder>thumb_label</zorder>
   <zorder>change_thumbnail</zorder>
   <zorder>description_text</zorder>
   <zorder>error_label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
